ecosystem/sep-0011.md

```diff
@@ -168,16 +170,24 @@ Enums are represented by the bare keyword of the value.  They can also be specif
 
 A few aggregate values are special-cased:
 
-##### Asset
+##### Asset, TrustLineAsset, ChangeTrustAsset
```

Comment:
Could you elaborate on this?

> We can't go from str -> XDR for the `ChangeTrustAsset` case

What do we need other than the code and the issuer account ID? Noting that, for the credit asset case, we can select whether it's a 4 or 12 character code using the logic defined on line 183.

Reply:
Great point. Thanks for catching this. I've pushed a changes that define encoding of AlphaNum4/12 separately so that for `ChangeTrustAsset` we string encode the fields holding the AlphaNum4 and AlphaNum12 appropriately. In regards to how this impact the SDKs, I think just as you mentioned, any function like `ChangeTrustAsset.fromCanonicalString(...)` cannot support liquidity pool shares. Folks need to construct that manually on their own.